src/lib/hooks/machine-status.svelte.ts

```diff
@@ -80,104 +67,21 @@ const sortResourceStatuses = (machineStatus: MachineStatus) => {
   };
 };
 
-export const provideMachineStatusContext = (refetchInterval: () => number) => {
```

Comment:
The provider for this hook and `useResourceNames` were causing more problems than they were worth, especially with the Tanstack 6 upgrade. Tanstack's query deduping and caching should be good enough to handle resource management when these hooks are called a large number of times.